config/database.yml

```diff
@@ -5,21 +5,26 @@
 #   gem "sqlite3"
 #
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
+  encoding: <%= ENV.fetch("RAILS_DB_ENCODING") { 'unicode' } %>
+  username: <%= ENV.fetch("RAILS_DB_USERNAME") { 'postgres' } %>
+  password: <%= ENV.fetch("RAILS_DB_PASSWORD") { 'postgres' } %>
+  host: <%= ENV.fetch("RAILS_DB_HOST") { 'localhost' } %>
+  port: <%= ENV.fetch("RAILS_DB_PORT") { 5432 } %>
 
 development:
   <<: *default
-  database: storage/development.sqlite3
+  database: appointment_scheduler_development
```

Comment:
IMHO, database name also should be in the env file too bhai

Reply:
@asis-wtag Thanks for the suggestions, bhai.